src/vtol.py

```diff
@@ -184,9 +157,71 @@ def change_status(self, new_status):
             raise Exception("Error: Unsupported status for vehicle")
         self.status = new_status
 
-    def include_heading(self):
-        '''Includes heading in messages'''
-        self.heading = True
+
+    def send_attitude_target(
+            self,
+            roll_angle=0.0,
+            pitch_angle=0.0,
+            yaw_angle=None,
+            yaw_rate=0.0,
+            use_yaw_rate=False,
+            thrust=0.5,
+    ):
+        '''
+        use_yaw_rate: the yaw can be controlled using yaw_angle OR yaw_rate.
+                    When one is used, the other is ignored by Ardupilot.
+        thrust: 0 <= thrust <= 1, as a fraction of maximum vertical thrust.
+                Note that as of Copter 3.5, thrust = 0.5 triggers a special case in
```

Comment:
I don't know if this in in our control or not. But is using a number in the range of possible thrusts really the best practice? additionally, why is thrust hard coded to 0.5? and if thrust is assigned to a variable number in between 0 and 1 what range would the hover mode activate in?

Reply:
This is also determined my ardupilot, the command just takes a thrust between 0 and 1. We can experiment with what values to use as we go i guess... I agree that this metric is not particularly informative. We would most likely use PID for this so it doesn't matter